Makefile.am

```diff
@@ -261,15 +261,16 @@ AM_CPPFLAGS = \
 
 AM_LDFLAGS = -Wl,--as-needed
 
-LOG_DRIVER = $(top_srcdir)/tools/tap-driver
-LOG_COMPILER = $(top_srcdir)/tools/tap-gtester
+PYTHON = $(shell which python3 2>/dev/null || which python)
```

Comment:
can't we turn this into a configure-time check instead of doing the shell-out constantly?

Reply:
Sounds good, I'll do that.

Reply:
Done, it uses automake's official `AM_PATH_PYTHON` now.